common/lib/xmodule/xmodule/modulestore/__init__.py

```diff
@@ -197,12 +197,14 @@ def _get_bulk_ops_record(self, course_key, ignore_case=False):
         # Retrieve the bulk record based on matching org/course/run (possibly ignoring case)
         if ignore_case:
             for key, record in self._active_bulk_ops.records.iteritems():
-                if (
+                # Shortcut: check basic equivalence for cases where org/course/run might be None.
+                if key == course_key or (
```

Comment:
@nasthagiri @ormsbee I discovered a fixe for the flaky test. In line 201 of common/lib/xmodule/xmodule/modulestore/split_mongo/split.py we are explicitly adding a bulk record for a course key with org, course, and run set to `None`. I'm not certain why the test was flaky--I estimate a 10% chance of false-negative--but this shortcut should circumvent the `AttributeError`.